.circleci/config.yml

```diff
@@ -527,6 +527,41 @@ jobs:
           command:
             python3 CircleciScripts/build_iossample.py "$(pwd)/aws-sdk-ios-samples"
   
+  release_xcframeworks:
+    macos:
+      xcode: "12.3.0"
+    steps:
+      - skip_job_unless_required
+      - checkout_with_cache
+      - pre_start_simulator
+      - restore_cache:
+          key: build_xcframeworks-result-{{ .Revision }}
+      - run:
+          name: Clone spm github repo
+          command: |
+            git clone "https://${GITHUB_SPM_RELEASE_TOKEN}@github.com/aws-amplify/aws-sdk-ios-spm.git"
+            cd aws-sdk-ios-spm
+            git checkout -b pre-release-${release_version}
+            cd ..
+      - run:
+          name: Prepare xcframework archives
+          command:
+            python3 CircleciScripts/prepare_xcframework_archives.py $release_version
+      - run:
+          name: Upload xcframework archives
+          command:
+            echo "Upload"
+            python3 CircleciScripts/upload_xcframeworks.py $release_bucket $release_version sdk_xcf_release
+      - run:
+          name: Create PR for SPM package manifest
+          command: |
+            git config --local user.name "${GITHUB_SPM_RELEASE_USER}"
+            cd aws-sdk-ios-spm
+            git add Package.swift
```

Comment:
What process modifies this to be updated with the latest version? I'm expecting a version update script.

Reply:
It happens in the step above "Prepare xcframework archives", inside the script - CircleciScripts/prepare_xcframework_archives.py [here](https://github.com/aws-amplify/aws-sdk-ios/blob/cf1245bf17f5ad8717f7eb0458f05d6e15b4f5ad/CircleciScripts/prepare_xcframework_archives.py#L57)